app/forms/totp_verification_form.rb

```diff
@@ -17,7 +18,7 @@ def submit
   attr_reader :user, :code
 
   def valid_totp_code?
-    return false unless code.match? pattern_matching_totp_code_format
+    return unless code.match? pattern_matching_totp_code_format
     Db::AuthAppConfiguration::Authenticate.call(user, code)
   end
```

Comment:
now that `Db::AuthAppConfiguration::Authenticate.call` returns a config object, can we rename this so it's no longer a predicate method that implies true/false? Mabe make it `otp_confp` or somthing

Reply:
good idea